style.css

```diff
@@ -0,0 +1,719 @@
+body {
+	margin: 0;
+	font-family: Montserrat;
+}
+.collection__img-mob {
+	display: none;
+}
+.header {
+	position: absolute;
+	z-index: 3;
+	width: 100%;
+}
+.header__inner {
+	display: flex;
+	justify-content: space-between;
+}
+.header__logo {
+	font-size: 30px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/step3/pullRequest?prid=5598887)